reason-react-native/src/apis/LayoutAnimation.re

```diff
@@ -0,0 +1,67 @@
+type animationConfig;
+[@bs.obj]
+external animationConfig:
+  (
+    ~duration: float=?,
+    ~delay: float=?,
+    ~springDamping: float=?,
+    ~initialVelocity: float=?,
+    ~_type: [@bs.string] [
+              | `spring
+              | `linear
+              | `easeInEaseOut
+              | `easeIn
+              | `easeOut
+              | `keyboard
+            ]
+              =?,
+    ~property: [@bs.string] [ | `opacity | `scaleX | `scaleY | `scaleXY]=?,
+    unit
+  ) =>
+  animationConfig =
+  "";
+
+type layoutAnimationConfig;
+[@bs.obj]
+external layoutAnimationConfig:
+  (
+    ~duration: float,
+    ~create: animationConfig=?,
+    ~update: animationConfig=?,
+    ~delete: animationConfig=?,
+    unit
+  ) =>
+  layoutAnimationConfig =
+  "";
+
+[@bs.module "react-native"] [@bs.scope "LayoutAnimation"]
+external configureNext:
```

Comment:
Maybe we should have two bindings `configureNext` and `configureNextWithEndCallback`?

Reply:
Further to your comment elsewhere regarding searching for such sister functions, perhaps we should simply add comments indicating that we have a convenience function whenever we do something like this?

Reply:
Yes, why not, I'd be happy to approve a (separate) PR adding such comments to all the xxxWithYYY functions. 👍